src/components/build-query-button.tsx

```diff
@@ -0,0 +1,52 @@
+import { Center, Tooltip, Button } from '@mantine/core';
+import Link from 'next/link';
+import { SupportedExportTypes } from './export-type';
+
+export interface BuildQueryButtonProps {
+  queryId: string | null;
+  dropdown: SupportedExportTypes;
+}
+
+/*
+ *  Component for the build query button at the bottom of each export-type component
+ *  navigates to the next page, and creates the query string
+ *
+ */
+export default function BuildQueryButton({ queryId, dropdown }: BuildQueryButtonProps) {
+  if (dropdown === 'system') {
```

Comment:
I think for readability, having this as a ternary operator in the main return would make more sense. So something like..

```
return (
    <>
         {dropdown === 'system' ? ( <Center> ...) : (<Center> ...) }
    </>  
```